src/types/types.ts

```diff
@@ -1584,6 +1584,48 @@ export interface SendPrivateTransactionOptions {
   fast: boolean;
 }
 
+export enum AssetType {
```

Comment:
naming: prefix some of these types with `Simulation`? or `Simulate`? otherwise we risk name clashing with other type exports

Reply:
prefixed with `Simulate`